cpp/src/arrow/compute/kernels/vector_sort_test.cc

```diff
@@ -209,138 +214,155 @@ TYPED_TEST(TestNthToIndicesRandom, RandomValues) {
 using arrow::internal::checked_pointer_cast;
 
 template <typename ArrowType>
-class TestSortToIndicesKernel : public TestBase {
+class TestArraySortIndicesKernel : public TestBase {
```

Comment:
This hunk is for renaming `SortToIndices()` -> `SortIndices()`. No logic change.